docs/src/components/header/HeaderNavBar.tsx

```diff
@@ -310,7 +310,7 @@ export default function HeaderNavBar() {
                     </li>
                     <li>
                       <ProductSubMenu
-                        id={PRODUCT_IDS[4]}
```

Comment:
https://github.com/mui/material-ui/blob/692fc468247229148f4d38f1d4ea3a37e6bfbc33/docs/src/components/header/HeaderNavBar.tsx#L68

We’re trying to access the 5th element of the `PRODUCT_IDS` array, but it only contains 4 items - that’s why the app is crashing.